mindmeld/models/taggers/pytorch_crf.py

```diff
@@ -0,0 +1,407 @@
+import logging
+import os
+import random
+import uuid
+from collections import Counter
+from copy import copy
+from itertools import zip_longest
+from random import randint
+
+import numpy as np
+import torch
+import torch.nn as nn
+from sklearn.feature_extraction import DictVectorizer, FeatureHasher
+from sklearn.metrics import f1_score
+from sklearn.model_selection import train_test_split
+from sklearn.preprocessing import LabelEncoder
+from torch import optim
+from torch.utils.data import Dataset, DataLoader
+from torchcrf import CRF
+
+from ...exceptions import MindMeldError
+from ...path import USER_CONFIG_DIR
+
+logger = logging.getLogger(__name__)
+
+DEFAULT_PYTORCH_CRF_ER_CONFIG = {
+    "feat_type": "hash",  # ["hash", "dict"]
+    "feat_num": 50000,
+    "stratify": True,
+    "drop_input": 0.2,
+    "train_batch_size": 8,
+    "patience": 3,
+    "epochs": 100,
+    "train_dev_split": 0.15,
```

Comment:
similar nit: Can we rename to `train_val_split`? Don't want the dev set here to be confused with dev set files you can add as part of the hierarchy. Can be persuaded otherwise though :)

Reply:
Renamed to train_val _split. I do think that would be the better recognized variable name.

Reply:
Just a thought: It can be easier for MindMeld users/developers if params terminology is similar across different modules where pytorch is being used (see [params](https://www.mindmeld.com/docs/userguide/deep_neural_nets.html#common-configurable-params) used in deep NNs docs). Things like epochs/number_of_epochs, train_val_split/dev_split_ratio, train_batch_size/batch_size, etc.